comment_iowriter.go

```diff
@@ -142,3 +144,135 @@ func (cw *RDJSONCommentWriter) Flush(_ context.Context) error {
 	}
 	return nil
 }
+
+var _ CommentService = &SARIFCommentWriter{}
+
+// SARIFCommentWriter
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>comment on exported type SARIFCommentWriter should be of the form "SARIFCommentWriter ..." (with optional leading article)
<!-- __reviewdog__:ChBlNjEzNjRiNzgwNjc3Y2NiEg9nb2xpbnQtY2lyY2xlY2k= -->